src/js/utils/dom.js

```diff
@@ -375,7 +375,7 @@ export function getAttributes(tag) {
   // known boolean attributes
   // we can check for matching boolean properties, but older browsers
   // won't know about HTML5 boolean attributes that we still read from
-  const knownBooleans = ',' + 'autoplay,controls,loop,muted,default' + ',';
+  const knownBooleans = ',' + 'autoplay,controls,playsinline,loop,muted,default' + ',';
```

Comment:
I think this is supposed to be `default` because `<track>` els use a `default` attribute. Going to add `defaultMuted` in here.